cli_tests/install_kubectl.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.20.1/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
This seems like a pretty long series of commands for just one line. Doing something like
```
curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.20.1/bin/linux/amd64/kubectl && \
chmod +x kubectl && \
sudo mv kubectl /usr/local/bin/
```
might make it a bit easier to read.

Reply:
Good call. Legible-ified.